Assets/Scripts/NPCs/NPCsGlobalVariables.cs

```diff
@@ -0,0 +1,24 @@
+﻿using UnityEngine;
+public class NPCsGlobalVariables
```

Comment:
All these variables could be stored in Indestructibles.cs, where we store all our other global variables. It would be consistent with the rest of the code and we wouldn't need this extra file